spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -41,34 +37,58 @@ MicroProfile applications that make use of APIs or versions not mentioned below
  - <<mp-opentracing, MicroProfile OpenTracing 2.0>>
  - <<mp-rest-client, MicroProfile Rest Client 2.0>>
 
+A MicroProfile compatible implementation must pass *all* the TCKs provided by the MicroProfile individual
+specifications. The Jakarta specifications TCKs are optional.
+
+These APIs and versions guarantee application developers a minimum set of APIs and are not intended to be restrictive.
+
 [[javase]]
-=== Java SE 8
+=== Java SE
 
 Libraries in use by the MicroProfile leverage new features provided in Java SE 8, therefore Java SE 7 and before are unsupported.
-MicroProfile 4.x implementations may provide support for Java SE 9 and beyond, however Java SE 8 compatibility must be maintained.
+MicroProfile implementations are free to provide support for any Java SE 8 and beyond.
 
 [[jakarta-cdi]]
 === Jakarta Contexts and Dependency Injection (CDI) 2.0
 
-Jakarta CDI provides the base for a growing number of APIs included in MicroProfile 4.0.
+Jakarta CDI provides the base for a growing number of APIs included in MicroProfile.
 Use of implementations beyond CDI 2.0 in MicroProfile is allowed, however, not required.
 
  - https://jakarta.ee/specifications/cdi/2.0/cdi-spec-2.0.html
  - http://cdi-spec.org/
 
+==== Optional Requirements
+
+The following CDI features are optional:
```

Comment:
Since we don't have the complete list on what are optional, I suggest instead of highlighting the list, we should just say "Some of CDI features are not required, such as EJB integration, Portable Extension, Expression langages etc.". Please refer to the individual implmentation on their support statement.